src/features/bookings/pages/BookingDetails/BookingDetails.tsx

```diff
@@ -132,7 +134,10 @@ export function BookingDetails() {
         },
       })
     }
-    throw new ScreenError(`Booking #${params.id} not found`, { Screen: BookingNotFound })
+    throw new ScreenError(`Booking #${params.id} not found`, {
+      Screen: BookingNotFound,
+      shouldBeCapturedAsInfo: shouldLogInfo,
+    })
```

Comment:
si la remote config est activée, on log une info
si la remote config est désactivée, on log une erreur

est-ce que je comprends bien ?

Reply:
ce mécanisme est utilisé à d'autres endroits

j'ai un gros doute, je ne suis pas certain qu'on veuille ce comportement

Reply:
`export class MonitoringError extends Error {
  constructor(
    message: string,
    { name, captureContext, skipLogging = false, shouldBeCapturedAsInfo = false }: ErrorInfo = {}
  ) {
    super(message)
    if (name) {
      this.name = name
    }

    if (shouldBeCapturedAsInfo) {
      eventMonitoring.captureException(this.message, { level: 'info' })
    }

    if (!skipLogging && !shouldBeCapturedAsInfo) {
      eventMonitoring.captureException(this, captureContext)
    }
  }
}`

Dans le cas des `ScreenError` `skipLogging` est à true donc tu n'auras aucun log